api/en/202_List_Role.md

```diff
@@ -133,73 +133,85 @@ The response is a JSON object, the correspondence between the key (name) and typ
     "results": [
       {
         "__metadata": {
-          "uri": "{CellURL}__ctl/Role(Name='{RoleName}',_Box.Name='{BoxName}')",
+          "uri": "https://cell1.unit1.example/__ctl/Role(Name='role1',
+_Box.Name='box1')",
           "etag": "W/\"1-1486349783744\"",
           "type": "CellCtl.Role"
         },
-        "Name": "{RoleName}",
-        "_Box.Name": "{BoxName}",
+        "Name": "role1",
+        "_Box.Name": "box1",
         "__published": "/Date(1486349783744)/",
         "__updated": "/Date(1486349783744)/",
         "_Box": {
           "__deferred": {
-            "uri": "{CellURL}__ctl/Role(Name='{RoleName}',_Box.Name='{BoxName}')/_Box"
+            "uri": "https://cell1.unit1.example/__ctl/Role(Name='role1',
+_Box.Name='box1')/_Box"

```

Comment:
Since we have copied the Japanese sample as it is to the English version with this modification, a line feed has been added.
There is no problem because it is unified.